app/controllers/programs_controller.rb

```diff
@@ -67,9 +70,16 @@ def create
 
   # PATCH/PUT /programs/1 or /programs/1.json
   def update
+    uniqname = program_params[:instructor_attributes][:uniqname]
+    if ProgramManager.find_by(uniqname: uniqname).present?
```

Comment:
Still is coupled to the ProgramManager.
I would like to work though the "decoupling" of this. I will create a JIRA ticket to address this